mindmaps-dashboard/src/js/EngineClient.js

```diff
@@ -63,61 +63,51 @@ export default class EngineClient {
      * Query Engine for concepts by type.
      */
     conceptsByType(type, fn) {
-        var request = {
+        this.request({
             url: "/graph/concept/"+type,
```

Comment:
I think engine has similar hard coded strings in the `io.mindmaps.util.REST` class. I am just wondering if there is a way for us to get all of these strings in one place. That way if we change end points we don't accidentally break everything.